saleor/graphql/discount/mutations.py

```diff
@@ -358,6 +365,24 @@ def clean_prices(cls, cleaned_input, voucher, errors):
                         cleaned_channel["channel_id"]
                     )
 
+    @classmethod
+    def clean_discount_values(cls, cleaned_input, voucher, errors):
+        channels_without_value = []
+        channels_with_invalid_value_precision = []
+        channels_with_invalid_percentage_value = []
+        channels_with_invalid_min_amount_spent_precision = []
+        error_lists = (
```

Comment:
If you would define error_list as an error_dict, then you wouldn't need the definition of the list from line 370-373 and mapping from 323 to 327